min/lib/CSSmin.php

```diff
@@ -765,7 +765,9 @@ private function str_slice($str, $start = 0, $end = FALSE)
     private function normalize_int($size)
     {
         if (is_string($size)) {
-            switch (substr($size, -1)) {
+            $letter = substr($size, -1);
+            $size = intval($size);
```

Comment:
The function call [adds a single return opcode](https://3v4l.org/rp2kS/vld#output). File under micro-optimization; not worth changing.